scripts/mongo/createViews.php

```diff
@@ -93,8 +93,8 @@ function generateViews($id, $viewId, $storeName, $stat)
         MongoCursor::$timeout = -1;
 
         print "Generating $viewId";
-        $tripod = new MongoTripod($viewSpec['from'], $storeName, array('stat'=>$stat));
-        $views = $tripod->getTripodViews();//new MongoTripodViews($tripod->storeName,$tripod->collection,$tripod->defaultContext);
+        $tripod = new \Tripod\Mongo\Tripod($viewSpec['from'], $storeName, array('stat'=>$stat));
```

Comment:
Not sure about `\Tripod\Mongo\Tripod` - the one above which is ok `\Tripod\Mongo\Config` - but now we've removed a level of class, but we've added a dummy one in? Should it just be `\Tripod\Mongo`